packages/core/src/components/AccordionGroup/Accordion.tsx

```diff
@@ -0,0 +1,31 @@
+import CardBase from '@components/Card/CardBase';
+import { AccordionProps } from './types';
+import { createWrapperStyles } from './styles';
+
+const Accordion = ({
+  title,
+  isActive,
+  ariaControls,
+  tabId,
```

Comment:
I suggest using `id` or `accordionId` or similar because it's not a tab.

Reply:
Done